### ISSUE TITLE
exception found in review.py during search un-used keywords

### ISSUE THREAD

**Issue by p74** [robotframework/RIDE#2068]
When press search un-used keywords then i got an exception:

Traceback (most recent call last):
  File "c:\program files (x86)\python37-32\lib\threading.py", line 917, in _bootstrap_inner
    self.run()
  File "c:\program files (x86)\python37-32\lib\threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "c:\program files (x86)\python37-32\lib\site-packages\robotide\ui\review.py", line 373, in _run
    if self._is_unused(keyword):
  File "c:\program files (x86)\python37-32\lib\site-packages\robotide\ui\review.py", line 381, in _is_unused
    self._controller.execute(FindUsages(keyword.name, keyword_info=keyword.info)).next()
AttributeError: 'generator' object has no attribute 'next'

If fix it as g.next() is not supported by python3

c:\program files (x86)\python37-32\lib\site-packages\robotide\ui\review.py", line 381,
change line from:
 self._controller.execute(FindUsages(keyword.name, keyword_info=keyword.info)).next()
to:
  next(self._controller.execute(FindUsages(keyword.name, keyword_info=keyword.info)))

Cheers Ruud

**Comment by Nyral** [comment]
Already fixed, see #2013

**Comment by HelioGuilherme66** [comment]
@p74 This is fixed in current development version. You can try by installing with:
```
pip install -U https://github.com/robotframework/RIDE/archive/master.zip
```
